libs/agno/agno/agent/agent.py

```diff
@@ -4475,6 +4506,55 @@ async def aprint_response(
                 panels = [p for p in panels if not isinstance(p, Status)]
                 live_log.update(Group(*panels))
 
+    def update_reasoning_content_from_tool_call(self, tool_name: str, tool_args: Dict[str, Any]) -> None:
```

Comment:
Shouldn't we handle the extra data here as well?

Reply:
yes, i left a comment to confirm that- https://github.com/agno-agi/agno/pull/2745#issuecomment-2790725787
doing the change